tests/python_tests/common.py

```diff
@@ -273,7 +273,7 @@ def run_continuous_batching(
     prompts: List[str],
     generation_configs : List[GenerationConfig]
 ) -> List[GenerationResult]:
-    pipe = ContinuousBatchingPipeline(model_path.absolute().as_posix(), scheduler_config, "CPU", {})
+    pipe = ContinuousBatchingPipeline(model_path.absolute().as_posix(), scheduler_config, "CPU", {}, {})
```

Comment:
why do we need to pass empty brackets? As I see we have a default value for this parameter

Reply:
The explanation is in https://github.com/openvinotoolkit/openvino.genai/pull/651#discussion_r1687871150